### ISSUE TITLE
Support for Chivalry 2

### ISSUE THREAD

**Issue by extnse** [Z1ni/XGP-save-extractor#39]
## Game name:  Chivalry 2
TLDR; Good day! Can you please add support for the save transfer for this great game? PC XboxGP version -> Steam version. 
Thanks for your time and effort!

I played Chivalry 2 quite a while on a PC XboxGamePass, and made some progress, which I want to save. But 99% of my games are on Steam, and I'm going to buy Chivalry 2 on Steam. Naturally, my progress will be lost. It will be extremely great, to be able to transfer the progress. And extremely sad, in case if it's not possible :D
So, thanks in advance, and best of luck!

## Game package name: TripwireInteractiveLLC.Chivalry2_237dqjpew1r3m

[wgs.zip](https://github.com/Z1ni/XGP-save-extractor/files/12774246/wgs.zip)

**Comment by Z1ni** [comment]
I searched around a bit, but I didn't find any save files for the Steam version, so I don't know if the save files are compatible or not.

If someone has the Steam version of the game, and can provide the files from `%LOCALAPPDATA%\Chivalry 2\Saved`, I could take a better look.

**Comment by extnse** [comment]
Okay, I got a Steam version and its saves.
[Saved.zip](https://github.com/Z1ni/XGP-save-extractor/files/13121713/Saved.zip)

**Comment by Z1ni** [comment]
It seems that the save format between the versions isn't the same, at least based on your files.

I'm afraid I can't do anything about this.
